pyproject.toml

```diff
@@ -97,7 +97,7 @@ exporters = [
 ]
 
 gui = [
-    "PySide6-Essentials>=6.5",
+    "PySide6-Essentials==6.9.1",
```

Comment:
why do we need to lock on the specific one?

Reply:
gotta at least to re-generate the ui files. That wasn't in the scope of my pr